corehq/apps/data_interfaces/migrations/0037_add_dedupe_update_toggle.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.toggles import CASE_DEDUPE, CASE_DEDUPE_UPDATES, NAMESPACE_DOMAIN
+from corehq.toggles.models import Toggle
+
+
+@skip_on_fresh_install
+def add_dedupe_update_toggle(apps, schema_editor):
+    for domain in CASE_DEDUPE.get_enabled_domains():
+        CASE_DEDUPE_UPDATES.set(domain, enabled=True, namespace=NAMESPACE_DOMAIN)
+
+
+def reverse(apps, schema_editor):
+    toggle = Toggle.cached_get(CASE_DEDUPE_UPDATES.slug)
+    toggle.delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_interfaces', '0036_backfill_dedupe_match_values'),
+    ]
+
+    operations = [
+        migrations.RunPython(add_dedupe_update_toggle, reverse_code=reverse),
```

Comment:
Could this operation be done in `corehq/apps/accounting/migrations/0089_dedupe_priv.py` to avoid two separate DB migrations?

Reply:
It could be, but I think I'd be looking to unwind this in the future. It feels awkward that the code to get the data_interfaces app in a workable state lives in the accounting app. Ideally, I think both of these migrations ultimately belong in data_interfaces (or even better, a dedicated dedupe app)